applications/nrf_desktop/src/modules/watchdog.c

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+ */
+
+#include <zephyr.h>
+#include <device.h>
+#include <drivers/watchdog.h>
+
+#define MODULE watchdog
+#include "module_state_event.h"
+#include "power_event.h"
+
+#include <logging/log.h>
+LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_WATCHDOG_LOG_LEVEL);
+
+
+#if defined(CONFIG_WDT_0_NAME)
+#define WATCHDOG_DEV_NAME CONFIG_WDT_0_NAME
+#elif defined(DT_WDT_0_NAME)
+#define WATCHDOG_DEV_NAME DT_WDT_0_NAME
+#else
+#error "Error!!! Watchdog device name not defined - verify configuration."
+#endif
+#define WDT_FEED_WORKER_DELAY 2000 //2s
```

Comment:
I thought about it, but current setting is set to be in seconds and relation to timeout value would end with division of timeout. That division is unnecessary in my opinion, because 2s seems to be good enought for shortest timeout (16s) as well as the longest possible.

Reply:
Ok